preview-env/comment/action.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: Create Deployment Result Summary
+description: |
+  Creates a deployment result summary as comment in the current PR when deployment deployment artifacts have been uploaded during this run.
+
+runs:
+  using: composite
+  steps:
+    - name: Download Deployment Status Artifacts
+      uses: actions/download-artifact@v4
+      with:
+        path: deployment_status
+        pattern: deployment-status-${{ github.run_id }}-${{ github.run_attempt }}-*
+
+    - name: Check for deployment artifacts
+      shell: bash
+      run: |
+        if [ -d deployment_status ] && find deployment_status -type f -name '*.md' | grep -q .; then
+          echo "artifacts_found=true" >> $GITHUB_ENV
+        else
+          echo "artifacts_found=false" >> $GITHUB_ENV
+        fi
```

Comment:
I would suggest to use proper outputs (`GITHUB_OUTPUT`) ? as the env variables defined there is never used as is.

It's too bad the actions/download-artifact@v4 action does not provide proper outputs :(